tests/test_api_organization.py

```diff
@@ -88,6 +103,58 @@ def test_it_returns_organization(self, json):
             actual = json
             assert expected == actual
 
+    class TestCreateKatalogusError(
+        UsesPostMethod,
+        UsesListEndpoint,
+        Returns500,
+    ):
+        data = static_fixture({"name": "Test Org 3", "code": "test3", "tags": ["tag2", "tag3"]})
+
+        @pytest.fixture(autouse=True)
+        def mock_services(self, mocker):
+            mocker.patch("katalogus.client.KATalogusClientV1.create_organization", side_effect=HTTPError("Test error"))
+            mocker.patch("octopoes.connector.octopoes.OctopoesAPIConnector.create_node")
+
+        def test_it_returns_error(self, json):
+            expected = {
+                "type": "server_error",
+                "errors": [
+                    {
+                        "code": "error",
+                        "detail": "Katalogus returned error creating organization: Test error",
+                        "attr": None,
+                    }
+                ],
+            }
+            assert expected == json
+
+    class TestCreateOctopoesError(
+        UsesPostMethod,
+        UsesListEndpoint,
+        Returns500,
+    ):
+        data = static_fixture({"name": "Test Org 3", "code": "test3", "tags": ["tag2", "tag3"]})
+
+        @pytest.fixture(autouse=True)
+        def mock_services(self, mocker):
+            mocker.patch("katalogus.client.KATalogusClientV1.create_organization")
+            mocker.patch(
+                "octopoes.connector.octopoes.OctopoesAPIConnector.create_node", side_effect=HTTPError("Test error")
+            )
+
+        def test_it_returns_error(self, json):
+            expected = {
+                "type": "server_error",
+                "errors": [
+                    {
+                        "code": "error",
+                        "detail": "Octopoes returned error creating organization: Test error",
+                        "attr": None,
+                    }
+                ],
+            }
+            assert expected == json
```

Comment:
The order in pytest is usually `assert actual == expected`, this way it assigns the correct value to 'actual' and 'expected' in the logs

```suggestion
            assert json == expected
```